apps/arweave/src/ar_http_iface_client.erl

```diff
@@ -630,6 +630,74 @@ cm_publish_send(Peer, Solution) ->
 		body => ar_serialize:jsonify({Json})
 	})).
 
+%% @doc Fetch the jobs from the pool proxy or coordinated mining exit peer.
+get_jobs(PeerOrURL, PrevOutput, Pool) ->
```

Comment:
What does `Pool` refer to here?

Reply:
It is a boolean that is true when the request is to be made to the pool. The name is confusing - should I rename it to `IsPoolRequest` or do you prefer to approach it differently?

Reply:
Hm... I guess this is similar to the other discussion... my preference would be to always query jobs from the Pool and not go through the CM Exit Node. The idea would be that it will simplify the code by eliminating all these dual codepaths - if after launch we find that the network traffic of the pool is a problem we can tackle that as a specific and holistic optimization (and the solution we find at that time could be something completely different and might not involve using the Exit node a proxy for the pool)

Reply:
Yep, I see your point. However, I was wrong - it was not the traffic that was our main concern; see @vird's comment [here](https://github.com/ArweaveTeam/arweave/pull/515/files#r1448529287).

Reply:
Got it. I've added a comment to clarify my understanding over there. It does sound like the dual path system is required to support CM and Pools. With that in mind I do think we'll want to give some thought to how to remove duplication and complexity... but I think that can wait until this first pass is done.

I've added an issue to flag this for later: https://github.com/ArweaveTeam/arweave-dev/issues/354

I realize adding an issue is the first step towards never actually cleaning the code up, but I also agree that it's no longer a priority (I had originally hoped we could remove the entire codepath, but I now see why it's necessary). I'll do my best to take a look sometime after you merge but before we release. 🤞

Reply:
As for the variable name here - how about reversing it and calling it `GetJobsFromExitNode` or something like that?